contracts/voting-verifier/src/events.rs

```diff
@@ -136,12 +136,16 @@ impl From<PollStarted> for Event {
 
 #[cw_serde]
 pub struct VerifierSetConfirmation {
+    #[deprecated(since = "1.1.0", note = "use message_id field instead")]
     pub tx_id: nonempty::String,
+    #[deprecated(since = "1.1.0", note = "use message_id field instead")]
```

Comment:
```suggestion
    #[deprecated(since = "1.3.0", note = "use message_id field instead")]
    pub tx_id: nonempty::String,
    #[deprecated(since = "1.3.0", note = "use message_id field instead")]
```
deprecated since the upcoming release